src/components/sections/home/GetStarted.tsx

```diff
@@ -95,9 +97,26 @@ const GetStartedSteps: React.FC = () => {
 
 const GetStarted: React.FC = () => {
   const [yamlLink, setYamlLink] = useState(
-    "https://litmuschaos.github.io/litmus/litmus-operator-v1.11.2.yaml"
+    "https://litmuschaos.github.io/litmus/litmus-operator-v1.9.0.yaml"
   );
-  const initialValue = { yamlLink: yamlLink, setYamlLink: setYamlLink };
+  const [latestVersion, setLatestVersion] = useState<string>("");
+
+  useEffect(() => {
+    fetch("https://hub.litmuschaos.io/api/version")
+      .then((response) => response.json())
+      .then((data) => {
+        setLatestVersion(data[0]);
+        setYamlLink(
```

Comment:
@S-ayanide , this reflects the experiments version - on the hub. While it *mostly* matches the operator version, the experiment (litmus-go) is more prone to patch releases than the operator. Typically we'd prefer using the monthly point releases for the chaos operator (control plane for experiments) unless explicitly recommended. 

This is great for the experiments link in Line no 84. 

Can we pattern match for the latest `x.x.0` release in https://hub.litmuschaos.io/api/version for the operator manifest if possible?

Reply:
Alright if I get it right, the change we need to make is replacing the last digit as 0 for the operator and experiment can remain the same with patch release version or it can be the monthly version also ending with a 0
Operator version : 1.12.0
Experiment version: 1.12.0 or 1.12.x

Is this understanding correct
@ksatchit

Reply:
I guess the since the monthly releases are more stable versions they can accommodate both for Operator as well as Experiment versions in the yaml link.